storage/client/README.md

```diff
@@ -0,0 +1,365 @@
+# Azure Storage Queue client library for Java
+Azure Queue storage is a service for storing large numbers of messages that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS. 
+A single queue message can be up to 64 KB in size, and a queue can contain millions of messages, up to the total capacity limit of a storage account.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation] | [Product documentation][storage_docs]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Create Strorage Account][storage_account]
+
+### Adding the package to your product
+
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-storage</artifactId>
+  <version>12.0.0</version>
+</dependency>
+```
+
+### Create a Storage Account
+To create a Storage Account you can use the Azure Portal or [Azure CLI][azure_cli].
+
+```Powershell
+az group create \
+    --name storage-resource-group \
+    --location westus
+```
+
+### Authenticate the client
```

Comment:
The clients are also able to use a SharedKey authentication which uses the storage account's connection string instead of a SAS token.

Reply:
Done.